### PR TITLE
[DOC] OpenMS Developer Quickstart Guide

### PR DESCRIPTION
**Pull request by oliveralka**
OpenMS Developer Quickstart Guide 

html output and latex output are working.

There is one issue with the table of content in the latex document.
The sections will be displayed as 0.1 Introduction instead of 1. Introduction.
I was not able to fix this, maybe someone has a suggestion. 

#3117

### CODE REVIEW

**Comment by jpfeuffer** [comment]
Were you able to get subsections in the LaTeX now?

**Comment by oliveralka** [comment]
@jpfeuffer: The subsections are shown, just the numbering is off (in the table of content). Everything seems just to have a 0. (so one more level of depth). 
I'm not sure where it comes from, since the format {article} - should have the depth:
- section 1. Introduction
   - subsection 1.1 General Information
right now it looks like:
- section 0.1. Introduction
  - subsection 0.1.1. General Information
That is the only issue.

**Comment by oliveralka** [comment]
Fixed the issue thanks to @timosachsenberg  and @jpfeuffer for the help.

File: doc/OpenMS_tutorial/OpenMS_Tutorial.doxygen [hunk 1]
```diff
@@ -247,7 +247,7 @@ General:
 </table >
 <br>
 <table>
-<caption id="multi_row">Directory strucrture of core library (/src/openms)</caption>
+<caption>Directory strucrture of core library (/src/openms)</caption>
```
Comment (jpfeuffer) [inline doc/OpenMS_tutorial/OpenMS_Tutorial.doxygen]: Spelling "structure"

File: doc/OpenMS_tutorial/OpenMS_Tutorial.doxygen [hunk 2]
```diff
@@ -303,7 +303,7 @@ To store identified peptides %OpenMS has classes
 
 <br>
 <table>
-<caption id="multi_row">Directory strucrture of core library (/src/openms)</caption>
+<caption>Directory strucrture of core library (/src/openms)</caption>
```
Comment (jpfeuffer) [inline doc/OpenMS_tutorial/OpenMS_Tutorial.doxygen]: Spelling "structure"

File: doc/OpenMS_tutorial/OpenMS_Tutorial.doxygen [hunk 3]
```diff
@@ -608,7 +608,7 @@ Now that we assembled the set of all protein accessions we are ready to compare
 <b>Example: Tutorial_Functionality.cpp</b>
 <br>
 Add method functionality
-\snippet Tutorial_Functionality.cpp Functionality_2
+\snippet Tutorial_Final.cpp Functionality_2
```
Comment (jpfeuffer) [inline doc/OpenMS_tutorial/OpenMS_Tutorial.doxygen]: Copy paste error?

Comment (oliveralka) [reply doc/OpenMS_tutorial/OpenMS_Tutorial.doxygen]: This should be Ok, take multiple snippets from the Tutorial_Final.cpp - The Example: Should be Tutorial_Final.cpp.

Comment (jpfeuffer) [reply doc/OpenMS_tutorial/OpenMS_Tutorial.doxygen]: Hm yes. I just want that readers easily find the right file to look up and run the snippet.
Btw: I think you added a "y" too much.

**Pull request merged**